src/Npgsql/NpgsqlConnector.cs

```diff
@@ -244,8 +244,7 @@ int ReceiveTimeout
 
         #endregion
 
-        internal NpgsqlDefaultDataReader DefaultDataReader { get; }
-        internal NpgsqlSequentialDataReader SequentialDataReader { get; }
+        internal NpgsqlDataReader DataReader { get; }
```

Comment:
That's strange, I always assumed the JIT generates identical code for field and auto-props... I did a small test, take a look at [this sharplabs.io disassembly](https://sharplab.io/#v2:EYLgZgpghgLgrgJwgZwLQDk4BstWFiAJQkiQDsBjCAFQE8AHFAGhhCmQFsAfAAQAYABDwCMAbgCwAKB4BmIQCYBAYSkBvKQM1C5PACwCA6ggCWMCAAUEAe0YIYtABQBKAeslaPQ4QE4HlmxB2tE4S7loAvhpaUZqyQvpGphAAYsYQWAAmzq4xniK+qekZIbmRYbE6woL+tvauAgDmEDCiAmUecUhQGVZkWLRegoWZoeFAA==) (or [this one for primitives](https://sharplab.io/#v2:EYLgZgpghgLgrgJwgZwLQDk4BstWFiAJQkiQDsBjCAFQE8AHFAGhhCmQFsAfAAQAYABDwCMAbgCwAKB4BmIQCYBAYSkBvKQM1C5PACwCA6ggCWMCAAUEAe0YIYtABQBKAeslaPQ4QE4HlmxB2tE4S7loAvhpaUZqyQvpGphAAYsYQWAAmzq4xniK+qekZIbmRYbFyxmQwAv629q4CAOYQMKICZR5xSFAZVmRYtAJVNYWZoeFAA==)). At least for these cases, the assembly seems to be identical whether you're referring to the field directly or via an auto-prop.

Do you have any idea about this?

PS Even if there's a difference, and we want to start using internal fields everywhere, I suggest that we deal with it in a separate PR, because it's a big change everywhere and not just for the reader.